test/cypress/integration/lazy-chat.test.js

```diff
@@ -0,0 +1,46 @@
+import { crispWebsiteId } from '../../../src/client/static/data/app.json'
+
+const lazyChatButtonIdentity = '[data-test="lazyChatButtonIdentity"]'
+const lazyChatOptInIdentity = '[data-test="lazyChatOptInIdentity"]'
+
+describe('Lazy chat', () => {
+  beforeEach(() => {
+    cy.visit('/')
+  })
+
+  it('is only present if Crisp ID is set', () => {
+    if (!!crispWebsiteId) {
+      cy.get(lazyChatButtonIdentity).should('be.visible')
+    } else {
+      cy.get(lazyChatButtonIdentity).should('not.exist')
+    }
+  })
+
+  it('asked me to accept the T&C when I hit the chat button ("terms_conditions" === false)', () => {
```

Comment:
Try to only use human language here, so something like:
"when I haven't accepted T&C yet, I'm prompted to accept the T&C when I hit the chat button"

(same for the following test)